CPC-CHARTER.md

```diff
@@ -237,27 +237,32 @@ CPC Chairperson or Director may serve.
 ## Section 8. Board Representation
 
 There are 3 Board seats earmarked for representation from the
-OpenJS Foundation projects.  The first of these board seats is filled by
-the CPC Director as outlined in Section 7 on [Elections][] and is
-always in place.  
+OpenJS Foundation projects.
 
-The second board seat requires 2 Platinum board members to activate
-and will be filled based on processes defined
+### CPC Director
+
+The first of these board seats is filled by the CPC Director as outlined in Section 7 on [Elections][] and is always in place.
+
+**CPC Director eligibility:** The CPC Director must be a current CPC Member
+
+**CPC Director voting:** CPC Members will vote on candidates
+
+### Community Director
+
+The second board seat will be filled based on processes defined
 by the CPC and approved by the Voting CPC members and then the Board. The
 CPC members will review and either re-confirm or update the
 processes once per year.
 
-For the first year after the formation of the OpenJS Foundation, the
-processes for filling this seat will be as follows:
+**Community Director eligibility:** The Community Director must be an [Active OpenJS Collaborator][] as defined in the CPC's Governance document.
 
-* As a constellation member of the OpenJS Foundation, the Node.js project
-  will elect one Board representative from either the Technical Steering Committee (TSC) or
-  Community Committee (CommComm). The representative will be selected according to a process
-  defined by the TSC and CommComm.
+**Community Director voting:** CPC Members will vote on candidates
```

Comment:
I struggled with this bit. I wanted to expand voting further to projects and potentially to the community but I'm not sure how to do that. In the end, I decided to keep it simple and the projects all have representation in the council.

Reply:
This should be explicit about whether it's everyone in the CPC or just its voting members that vote.